tests/h/util/uri_test.py

```diff
@@ -4,183 +4,302 @@
 
 from h.util import uri
 
-TEST_URLS = [
```

Comment:
All of these URLs are still here (minus a few duplicates), in the same order, but grouped into behaviors we are testing for. This makes the test output clearer as we will have a failing test like "test_it_strips_via_urls" rather than "test_normalise".